docker/entrypoint.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -x
+
+if [ ! -f /app/zebrad.toml ]; then
+echo "
+[consensus]
+checkpoint_sync = ${CHECKPOINT_SYNC}
+[metrics]
+endpoint_addr = 0.0.0.0:9999
+[network]
+network = ${NETWORK}
+[state]
+cache_dir = /zebrad-cache
+[tracing]
+force_use_color = true
+endpoint_addr = 0.0.0.0:3000" > /app/zebrad.toml
```

Comment:
Security Question: Does binding to `0.0.0.0` inside a docker container expose the port to the whole internet?

Reply:
It expose it to the host. If you configure the host with forwarding, then it also forwards to the internet, but that's not the default behavior.